pkg/runner/client.go

```diff
@@ -291,5 +252,5 @@ func getSchemaLease(ctx context.Context, conn *pgx.Conn) error {
 	if err != nil {
 		return err
 	}
-	return pgmodel.CheckSchemaVersion(ctx, conn, appVersion, false)
+	return pgmodel.CheckPromscaleExtInstalledVersion(conn)
```

Comment:
I think this call is no longer necessary as it's handled in extension.CheckVersions. Before we had a schema version separate from the extension versions check. We no longer have that.

Reply:
addressed in https://github.com/timescale/promscale/pull/1132/commits/52478ebb1321b62e360811d1e6c4abb0696922ee